### DISCUSSION TITLE
新的信息图真好看啊

### DISCUSSION THREAD

**Discussion by Xm798** [Arondight/Adachi-BOT#607] [General]
RT，感慨一下。
另外这样是不是可以把三命和五命加上…

**Comment by Arondight** [comment]
确实，赞美 @Mark9804 ，另外三五命不加

**Comment by mark9804** [comment]
很早之前我也提过这个不成熟的想法 [overview允许自定义关键命座，而不是固定显示一二四六](https://github.com/Arondight/Adachi-BOT/issues/217#issuecomment-941975250)，否决过程贴在链接里了

所以 overview 的显示逻辑是：我们默认查询者知道三五命是固定的 EQ 等级 +3，但是不知道其他信息。用户通过我们的图片提供的信息，和自己认知中的 “三五命是固定的 EQ 等级 +3” 结合，形成一个完整的 overview。

~~不过我还是觉得不一定有人一开始就知道三五命是固定提升就是了，比如我自己~~ 老穷鬼了

**Reply by Xm798** [reply]
三五命确实是 EQ 等级 +3，但问题在于这个提升**并不是固定的**，表现在有的角色三命加 E，有的角色三命加 Q。例如胡桃三命为`蝶引来生的技能等级提升3级`，绫华是`神里流•霜灭的技能等级提高3级`。原讨论串提到`三命和五命是固定的啊，都是大招+3`是不完全正确的，不同角色之间三五命的信息会有区别。

**Reply by mark9804** [reply]
那给已有的 json 文件加上三五命信息这个艰巨的任务就交给你了，组织看好你.jpg

加上并提交之后当前网页会显示错误的命座信息，这个到时候我来修

**Comment by Xm798** [comment]
@Mark9804 啊对，还有一件事。目前新的信息图会给数字换行，看起来不太易读，或许可以考虑优化一下？

![3e53bb587901865c.jpg](https://user-images.githubusercontent.com/48074861/150735339-c50ee212-1a18-49be-a74a-6dbad1d3a50d.jpg)

**Reply by mark9804** [reply]
大概现在已经不会强行分段了，看看
